### ISSUE TITLE
[Error -1]  

### ISSUE THREAD

**Issue by Darksaidx26** [PojavLauncherTeam/PojavLauncher#1944]
### Describe the bug

I downloaded version 1.14.2 and it tells me error -1, can someone tell me why this error is due?

### Steps To Reproduce

```markdown
1= download the version 1.14.2 
2= play the version 
3= click on the screen
4= get the error
```

### Expected Behavior

wait for the home screen to load, I touch the screen and I get the error, but if I do not touch the screen I do not get

### Platform

```markdown
- Device model: Samsung A21s / 32GB
- CPU architecture: aarch64
- Android version: 11
- PojavLauncher version: 3.3.1.1-1833-9680ba11bv3_openjdk
```

**Comment by HeinIsTaken** [comment]
Just do not click the screen

**Comment by wackycreeper** [comment]
What error message do you get? java.lang.ArrayIndexOutOfBoundsException? 
java.lang.StringIndexOutOfBoundsException?

**Comment by wackycreeper** [comment]
Also, send latestlog if possible?

**Comment by Darksaidx26** [comment]
---- Minecraft Crash Report ----
// There are four lights!

Time: 31/08/21 21:32
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at cud.u(MainWindow.java:339)
	at cud.c(MainWindow.java:382)
	at org.lwjgl.glfw.GLFWWindowSizeCallback$Container.invoke(GLFWWindowSizeCallback.java:80)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:1196)
	at cud.l(MainWindow.java:581)
	at cud.c(MainWindow.java:406)
	at cvk.b(SourceFile:1023)
	at cvk.e(SourceFile:976)
	at cvk.b(SourceFile:411)
	at net.minecraft.client.main.Main.main(SourceFile:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:159)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:30)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.14.2
	Operating System: Linux (aarch64) version Android-11
	Java Version: 1.8.0-internal, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 751626704 bytes (716 MB) / 1040187392 bytes (992 MB) up to 1040187392 bytes (992 MB)
	JVM Flags: 3 total; -Xms1017M -Xmx1017M -Xbootclasspath/p:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/ResConfHack.jar:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/cacio-androidnw-1.10-SNAPSHOT.jar:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/cacio-shared-1.10-SNAPSHOT.jar
	Launched Version: 1.14.2
	LWJGL: 3.2.3 SNAPSHOT
	OpenGL: GL4ES wrapper GL version 2.1 gl4es wrapper 1.1.4, ptitSeb
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	CPU: 8x null
	OptiFine Version: OptiFine_1.14.2_HD_U_F1
	OptiFine Build: 20210124-170418
	Render Distance Chunks: 2
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 2.1 gl4es wrapper 1.1.4
	OpenGlRenderer: GL4ES wrapper
	OpenGlVendor: ptitSeb
	CpuCount: 8

**Comment by Mathias-Boulay** [comment]
duplicate from #1809
